test/debugger.ts

```diff
@@ -116,43 +118,45 @@ export class Debugger extends common.ServiceObject {
    *     called with a list of Breakpoint objects as a parameter, or an Error
    *     object if an error occurred in obtaining it.
    */
-  listBreakpoints(debuggeeId: string, options: {
-    includeAllUsers: boolean;
-    includeInactive: boolean;
-    stripResults: boolean;
-    actions: string
-  }, callback: (err: Error|null, breakpoints?: stackdriver.Breakpoint[]) => void) {
-    if (typeof(options) === 'function') {
+  listBreakpoints(
+      debuggeeId: string, options: {
+        includeAllUsers: boolean; includeInactive: boolean;
```

Comment:
Nit: Should we move includeInactive down a line?

Reply:
`gts fix` made this change.